browser/page_mapping.go

```diff
@@ -423,70 +423,87 @@ func mapPage(vu moduleVU, p *common.Page) mapping { //nolint:gocognit,cyclop
 func mapPageOn(vu moduleVU, p *common.Page) func(common.PageOnEventName, sobek.Callable) error { //nolint:funlen
 	rt := vu.Runtime()
 
+	pageOnEvents := map[common.PageOnEventName]struct {
+		mapp func(vu moduleVU, event common.PageOnEvent) mapping
+		prep func() error
```

Comment:
I saw that `initiating` is used in the error if the prep function fails, and actually I quite like that over prepare. Maybe this can be:

```suggestion
		init func() error
```